Benchmarks/SystemSqliteDatabaseManager.cs

```diff
@@ -669,27 +669,27 @@ public static ConcurrentBag<WriteObject> GetMissingFromFirst(string firstRunId,
                 cmd.Parameters.AddWithValue("@first_run_id", firstRunId);
                 cmd.Parameters.AddWithValue("@second_run_id", secondRunId);
                 using (var reader = cmd.ExecuteReader())
-                while (reader.Read())
-                {
-                    var runId = reader["run_id"].ToString();
-                    var resultTypeString = reader["result_type"].ToString();
-                    if (runId != null && resultTypeString != null)
+                    while (reader.Read())
                     {
-                        var wo = WriteObject.FromBytes((byte[])reader["serialized"], (RESULT_TYPE)Enum.Parse(typeof(RESULT_TYPE), resultTypeString), runId);
-                        if (wo is WriteObject WO)
```

Comment:
Adding the comment inline to check if (reader.HasRows) or an outer loop while (reader.HasRows) is a good practice in case you do other things before reading see https://forums.asp.net/t/1175522.aspx?sqldatareader+Hasrows+vs+Read+

Reply:
Not needed here. I don't need to do anything before parsing the first row.